config.toml.example

```diff
@@ -309,7 +309,9 @@
 # Number of codegen units to use for each compiler invocation. A value of 0
 # means "the number of cores on this machine", and 1+ is passed through to the
 # compiler.
-#codegen-units = 1
+#
+# Uses the Cargo defaults: https://doc.rust-lang.org/cargo/reference/profiles.html#codegen-units
```

Comment:
To be clear, these are `rustc` defaults, not Cargo: https://doc.rust-lang.org/rustc/codegen-options/index.html#codegen-units

Reply:
I didn't know that, thanks :)